blocks/init/src/Blocks/components/accordion/accordion.php

```diff
@@ -53,33 +53,33 @@
 ?>
 
 <div
-	class="<?php echo \esc_attr($accordionClass); ?>"
-	data-accordion-open="<?php echo \esc_attr($accordionIsOpen ? 'true' : 'false'); ?>"
-	data-close-others="<?php echo \esc_attr($accordionCloseOthers ? 'true' : 'false'); ?>"
+	class="<?php echo esc_attr($accordionClass); ?>"
```

Comment:
This is odd tbh because the slevomat CS should enforce the fully qualified global functions and constants.

So if you remove the slashes, you should import them at the top with `use` statement.

The rationale is to optimize these calls.

Reply:
<img width="1669" alt="Screenshot 2022-03-10 at 15 22 04" src="https://user-images.githubusercontent.com/1742806/157681389-c9948c9e-a3fe-49a4-9840-4979b44b10d5.png">

Reply:
Oh totally missed that. Neat! Ignore me then XD